nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/FacetedTagService.java

```diff
@@ -47,15 +48,22 @@
  */
 public class FacetedTagService extends AbstractTagService {
 
+    protected static final String LABEL_PROPERTY = "label";
+
+    protected  static final String USERNAME_PROPERTY = "username";
```

Comment:
You can make these public, they could be useful to other code that needs to hit the tag internals directly.